src/components/payment/case-payment-history.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { Payment } from './payment.model';
+import { HttpService } from '../../services/http.service';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class CasePaymentHistoryService {
```

Comment:
I'm not 100% sure but I don't know whether we want to have Services in the toolkit with complex logic like http calls 
I was under the impression that the toolkit is just for display components. 
Perhaps we can have the service live in our management web and live the component + template here?

Reply:
If we have the service live in management web we would still have to pass it to component here, no?

Reply:
yes. The component here will require a service as dependency. Management web will be responsible to provide a service to retrieve payments. 
This way we can keep ui-toolkit simple and only dealing with ui stuff